index.md

```diff
@@ -10,7 +10,7 @@ Government web services should provide reliable content and be protected from im
 
 Plain HTTP (`http://`) connections are insecure, and allow an attacker to easily intercept traffic, track users, or impersonate the government.
 
-In today's internet, and today's browsers, HTTPS is an important baseline for _all_ government websites, no matter how small or how sensitive.
+With the internet and browsers of today, HTTPS is an important baseline for _all_ government websites, no matter how small or how sensitive.
```

Comment:
I like the other two changes, but I want to keep the original here. It's less compact, but I prefer it rhetorically.

That said, it's also likely that this topmost section is going to be replaced with a larger document soon. The sections below, which counter the "overkill" argument, are likelier to survive as their own page.